### DISCUSSION TITLE
Paypal/Patreon

### DISCUSSION THREAD

**Discussion by dozer1dave** [tmknight/docker-nordvpn#51] [General]
@tmknight, I find this container to be super useful and would like to help support it.  Do you have a Paypal/Patreon/Venmo you would be willing to share for those who wish to donate?  Thanks for your work to provide this and responsiveness with issues.

**Comment by tmknight** [comment]
Thank you for the offer; I'm glad to hear you are finding this project useful.  I look at this project as a community effort and for the benefit of all.  The care and feeding is recreation for me and requires no more payment than the continued use by the community and, hopefully, contribution back to the project by means of code submissions to fix/improve the experience and stability or simply ideas for improvement are welcomed.

Cheers!
